features/formatter/use_the_tap_formatter.feature

```diff
@@ -0,0 +1,87 @@
+Feature: Use the TAP formatter
+  So that I can get non-XML parseable results
+  As a Developer
+  I need to be able to use a TAP formatter
+
+  Scenario: Empty spec causes pending result
```

Comment:
That often happens initially with new features, then as the code grows they diverge more.